taxonomies/food/ingredients.txt

```diff
@@ -17252,16 +17282,21 @@ en: refined rapeseed oil
 hr: rafinirano ulje repice
 lt: rafinuotas rapsų aliejus
 pl: rafinowany olej rzepakowy, olej rzepakowy rafinowany
+ecobalyse:en: refined-rapeseed-oil
 
 < en:rapeseed oil
 en: british rapeseed oil
 de: Britisches Rapsöl
 hr: britanskog repičinog ulja
+ecobalyse:en: rapeseed-oil
```

Comment:
Can you leave only the ecobalyse properties for rapeseed oil on the parent en:rapeseed-oil and remove it from its children? (this one and more below)

Reply:
Noted ! It will be corrected in the next PR